corehq/util/test_utils.py

```diff
@@ -403,3 +403,23 @@ def create_test_case(domain, case_type, case_name, case_properties=None, drop_si
             CaseAccessorSQL.hard_delete_cases(domain, [case.case_id])
         else:
             case.delete()
+
+
+def set_parent_case(domain, child_case, parent_case):
```

Comment:
This is very similar to `corehq.apps.data_interfaces.tests.test_auto_case_updates.set_parent_case`